robi.py

```diff
@@ -1,14 +1,10 @@
-import discord
-from discord import Embed
-from discord.ext import commands
-from discord.ext.commands import ExtensionError
-import random
 import os
-import robiconf
 import robi_request
-from robiconf.bot_configuration import Token, BotInstance
+from discord.ext import commands
+import base64
 
-bot = BotInstance.bot #ngambil object bot dari robiconf
+bot = commands.Bot('!')
+token_utf8 = base64.b64decode('TnpJM09EWTJNRFV4TlRVNU1UWXhPVE0yLlh3Tk5BUS5ZSk8wWHlfMHV4MzEyMG5GMURYRml1SVQyLW8=')

```

Comment:
Is this safe?

Reply:
At least discord will not detect the Token now and now we don't need to extract the robiconf.rar